python/pyspark/ml/param/_shared_params_code_gen.py

```diff
@@ -133,7 +133,10 @@ def get$Name(self):
         ("thresholds", "Thresholds in multi-class classification to adjust the probability of " +
          "predicting each class. Array must have length equal to the number of classes, with " +
          "values >= 0. The class with largest value p/t is predicted, where p is the original " +
-         "probability of that class and t is the class' threshold.", None)]
+         "probability of that class and t is the class' threshold.", None),
+        ("weightCol", "weight column name. If this is not set or empty, we treat " +
+         "all instance weights as 1.0.", None)]
```

Comment:
So in one place in the template a "." is added and in another a "." is not. I've gone ahead and just removed period in the template (since it seems like the punctuation in the input is written without it).